src/components/product/OutOfStock/out-of-stock.scss

```diff
@@ -0,0 +1,39 @@
+@import "src/styles/scaffold";
+
+[data-store-out-of-stock] {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  margin-top: 24px;
+}
+
+[data-store-out-of-stock-input] {
+  width: 100%;
+  margin-top: var(--space-3);
+  margin-bottom: var(--space-3);
+  padding: var(--space-1) var(--space-7) var(--space-1) var(--space-2);
+  border: var(--border-width-0) solid var(--color-border-input);
+  border-radius: var(--border-radius-default);
+  transition: box-shadow .2s ease, border .2s ease;
+
+  &:hover {
+    border-color: var(--color-border-input-selected);
+    box-shadow: 0 0 0 var(--border-width-0) var(--color-border-input-selected);
+  }
+
+  @include input-focus-ring;
+}
+
+[data-store-out-of-stock-button] {
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  min-height: var(--tap-min-size);
+  padding: var(--space-1)  var(--space-3);
+  font-weight: var(--text-weight-bold);
+  text-decoration: none;
+  column-gap: var(--space-2);
```

Comment:
Try running `stylelint --fix` to sort these properties the right way.

Reply:
@sergiofontes I guess `stylelint --fix` runs every time we commit locally. I also tried running it manually, but nothing changed. What should be the correct order for these properties?

Reply:
Hm... that's odd, maybe it's not working properly, then. Anyway, the correct properties order [is in this link](https://github.com/stormwarning/stylelint-config-recess-order/blob/main/index.js).